search/views_test.py

```diff
@@ -0,0 +1,164 @@
+"""
+Tests for the search view
+"""
+from django.core.urlresolvers import reverse
+from django.db.models.signals import post_save
+from factory.django import mute_signals
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from courses.factories import ProgramFactory
+from dashboard.factories import ProgramEnrollmentFactory
+from profiles.factories import ProfileFactory, UserFactory
+from roles.models import Role
+from roles.roles import Staff
+from search.base import ESTestCase
+
+
+class SearchTests(ESTestCase, APITestCase):
+    """Tests for the search api view"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super(SearchTests, cls).setUpTestData()
+        # create some students
+        with mute_signals(post_save):
+            cls.students = [(ProfileFactory.create()).user for _ in range(30)]
```

Comment:
Just FYI the search queries below will only get the first 20 results. I don't think that matters in any of these tests though